python/tk_desktop/project_model.py

```diff
@@ -218,10 +217,7 @@ def __init__(self, parent, overlay_parent_widget):
         """ Constructor """
         ShotgunModel.__init__(self, parent, download_thumbs=True)
 
-        # override the default connection to use one that is tied to the current login
-        connection = ShotgunLogin.get_instance_for_namespace("tk-desktop").get_connection()
-
-        self.set_shotgun_connection(connection)
+        self.set_shotgun_connection(shotgun.create_sg_connection())
```

Comment:
It should probably reuse a thread-local one. I'll grab the sgtk.shotgun instance.